src/family_name_classifier.py

```diff
@@ -0,0 +1,315 @@
+import builtins
```

Comment:
I tried to do this code in PEP 8, but seems like PyCharm forgot to warn me about this  (or I ignored it).